devtools/gradle/src/main/java/io/quarkus/gradle/dependency/ConditionalDependenciesEnabler.java

```diff
@@ -194,14 +194,14 @@ private ExtensionDependency loadExtensionInfo(Path descriptorPath, ModuleVersion
         List<Dependency> conditionalDependencies = new ArrayList<>();
         if (extensionProperties.containsKey(BootstrapConstants.CONDITIONAL_DEPENDENCIES)) {
             String conditionalDeps = extensionProperties.get(BootstrapConstants.CONDITIONAL_DEPENDENCIES).toString();
-            for (String conditionalDep : conditionalDeps.split(",")) {
+            for (String conditionalDep : conditionalDeps.split(" ")) {
```

Comment:
```suggestion
            for (String conditionalDep : conditionalDeps.split("\\s+")) {
```

Reply:
I guess the whitespace character group makes sense but I'd suggest to precompile it instead of creating throwaway `Pattern`s:   https://www.baeldung.com/java-regex-pre-compile#benefits

Reply:
RIght, we should have a common utility in bootstrap for this. I'll do it later. Let's get it in, asap.